### ISSUE TITLE
Embed site screenshot on site information page

### ISSUE THREAD

**Issue by gwire** [openrightsgroup/blocked-org-uk#335]
A number of URLs covered by blocking orders no longer point to active sites.

For court-ordered blocks, it would be good if we could add a screenshot of the site on to the page, since the user may not be able to check directly.

Could this be done using a third-party service. Could we integrate with the urlscan.io api?

**Comment by edjw** [comment]
@gwire – Are you saying this should just be for court-ordered blocks?

**Comment by gwire** [comment]
No, just that it would be more useful for court-ordered blocks.
